mungegithub/mungers/sync/issue-sync.go

```diff
@@ -221,8 +248,53 @@ func (s *IssueSyncer) createIssue(source IssueSource) (issueNumber int, err erro
 		owner,
 	)
 	if err != nil {
-		return 0, err
+		return nil, err
 	}
 	glog.Infof("Created issue %v:\n%v", *obj.Issue.Number, body)
-	return *obj.Issue.Number, nil
+	return obj, nil
+}
+
+// autoPrioritize takes a flaky test issue filed by k8s-bot and prioritize it based on the number of flakes
+func (s *IssueSyncer) autoPrioritize(obj *github.MungeObject) error {
+	comments, err := obj.ListComments()
+	if err != nil {
+		return fmt.Errorf("Failed to list comment of issue")
+	}
+	count := 0
+	for _, c := range comments {
+		// "Failed:" is a special string contained in flake issue filed by k8s-bot
+		// Please make sure it matches the body generated by IssueSource
+		if !robotUser.Has(*c.User.Login) || !strings.Contains(*c.Body, "Failed:") {
```

Comment:
Maybe push this logic into each issue source? Especially that second condition could vary.